src/e2e-test/java/io/cdap/plugin/servicenow/actions/ServiceNowPropertiesPageActions.java

```diff
@@ -31,6 +33,8 @@
  */
 public class ServiceNowPropertiesPageActions {
 
+  public static Object clickaddConnectionButton;
```

Comment:
This is unused, remove it if we don't need it.

Reply:
done